README.md

```diff
@@ -59,6 +59,8 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
 ```
 
+**Note:** Loading `nvm` is temporarily slow. While we work on a fix, you can add `--no-use` to the end of the above script (...`nvm.sh --no-use`). This will not load nvm until you actually `use` it (which you'll have to do manually, e.g. `nvm use default`).
```

Comment:
I'd prefer to document the feature rather than what it's a workaround for. nvm isn't actually slow for most users; it's an unfortunate and vocal minority that experiences this.